### ISSUE TITLE
BNGParser doesn't throw error if rule specified with incorrect number of rate constants/laws

### ISSUE THREAD

**Issue by GoogleCodeExporter** [RuleWorld/bionetgen#66]
```
If a unimolecular rule is defined with two rate constants/laws then BNG throws 
the following error: "ERROR: Unidirection reaction may have only one rate law." 
Conversely, if a bimolecular rule is defined with one rate constant/law then 
BNG throws: "ERROR: No RateLaw supplied for Reaction Rule." The BNGParser does 
not recognize these errors. It would be nice if it did.
```

Original issue reported on code.google.com by `lh64@cornell.edu` on 4 Dec 2012 at 10:50
